pom.xml

```diff
@@ -784,17 +784,21 @@
             <artifactId>graphql-java</artifactId>
             <version>9.2</version>
         </dependency>
-        <!-- For RAPTOR routing -->
         <dependency>
-            <groupId>com.conveyal</groupId>
-            <artifactId>r5</artifactId>
-            <version>4.5.3</version>
-            <exclusions>
-                <exclusion>
-                    <artifactId>slf4j-simple</artifactId>
-                    <groupId>org.slf4j</groupId>
-                </exclusion>
-            </exclusions>
+            <groupId>org.apache.httpcomponents</groupId>
+            <artifactId>httpclient</artifactId>
+            <version>4.5.5</version>
+        </dependency>
+        <dependency>
+            <groupId>commons-cli</groupId>
+            <artifactId>commons-cli</artifactId>
+            <version>1.4</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>net.sourceforge.javacsv</groupId>
+            <artifactId>javacsv</artifactId>
+            <version>2.0</version>
```

Comment:
When removing r5 as a dependency I had to add the httpclient, commons-cli and javacsv. There are probably alternatives to these that we already have as a dependency. Java 11 have HTTP support. The jcommander may replace commons-cli, but I diden´t want to change to much. The  commons-cli is only a test dependency - used by the speed test only.